modules/Microsoft.Web/sites/.test/webAppCommon/deploy.test.bicep

```diff
@@ -153,5 +153,6 @@ module testDeployment '../../deploy.bicep' = {
     userAssignedIdentities: {
       '${nestedDependencies.outputs.managedIdentityResourceId}': {}
     }
+    basicPublishingCredentialsPolicyName: 'ftp'
```

Comment:
This is what I meant with array of objects
```suggestion
    basicPublishingCredentialsPolicies: [
      {
        name: 'ftp'
      }
      {
        name: 'scm'
      }
    ]
```

Reply:
Updated as per your suggestion